pytorch_lightning/__init__.py

```diff
@@ -37,10 +38,11 @@
 - https://pytorch-lightning.readthedocs.io/en/latest
 - https://pytorch-lightning.readthedocs.io/en/stable
 """
-
-_logger = python_logging.getLogger("lightning")
-_logger.addHandler(python_logging.StreamHandler())
-_logger.setLevel(python_logging.INFO)
+python_logging.basicConfig(
+    stream=sys.stdout,
```

Comment:
I think it is like a convention not to use stdout, but stderr for this?

Reply:
yes, I think that's reasonable.